benchmarks/manoptuna.jl

```diff
@@ -0,0 +1,215 @@
+using Manifolds
+using PythonCall
+include("benchmark_comparison.jl")
+
+# This script requires optuna to be available through PythonCall
```

Comment:
I have no experience with optuna, could this be documented a bit what the goal of this is?